templates/static_analysis/ios_binary_analysis.html

```diff
@@ -506,16 +506,40 @@ <h5 class="card-title"></h5>
                   <tbody>
                     {% if ats_analysis|length > 0 %}
                       {% for findings in ats_analysis %}
-                      <tr>
-                        <td>
-                         {{findings.issue}}
-                        <td>
-                          <span class="badge bg-danger"> {{findings.status}}</span>
-                        </td>
-                        <td>
-                         {{findings.description}}
-                        </td>
-                      </tr>
+                        {% if "insecure" in findings.status %}
```

Comment:
See my comments in `ios_report.html`

Reply:
This has been changed in the latest commit.